chingu-voyages/v18-bears-team-04
# Server setup

#### Description
This PR setup the server for the backend at Port 6000, create a user login and signup endpoints

#### Type of change
Setup a server for the application via the project root folder entry file App.js, create a user login and signup endpoint
- [x] setup project folders
- [x] add a server to the project
- [x] login endpoint
- [x] signup endpoint

#### How Has This Been Tested?
- clone this repo
- checkout to this branch by running `npm checkout ch-setup-server`
- start the `Server` using `npm run start:dev`
- you should see a message showing application is running at port 6000 and database 
   connection successful
-  goto postman and test the endpoint
- ensure you added .env and development.json files in your root folder
- run the test file using `npm test`


#### Checklist:

#### PT-ID
N/A
[](url)
N/A
#### Screenshots
![image](https://user-images.githubusercontent.com/33469121/78590380-a8e0df80-7839-11ea-8519-516bacebc5c3.png)
![image](https://user-images.githubusercontent.com/33469121/78960786-02ba0180-7ae7-11ea-861d-cc32e3c76c0b.png)
![image](https://user-images.githubusercontent.com/33469121/78960804-0f3e5a00-7ae7-11ea-931a-2be7d6063e9b.png)
![image](https://user-images.githubusercontent.com/33469121/78960850-3ac14480-7ae7-11ea-8e76-d378c277573b.png)


#### Questions:
